src/cramFile/file.js

```diff
@@ -44,6 +57,9 @@ export default class CramFile {
     this.featureCache = new LRU({
       maxSize: this.options.cacheSize,
     })
+    if (getEndianness() > 0) {
+      console.warn('Detected big-endian machine, may not be able to run')
```

Comment:
I think unless we verify that this does indeed work on big-endian machines, we should throw here. Message should be something like "cram-js does not support big-endian machine architectures" or similar